Client/src/components/Tooltips/TooltipWithTail/TooltipWithTail.tsx

```diff
@@ -0,0 +1,54 @@
+import "./tooltipWithTail.css";
+import React from "react";
+import { styled } from "@mui/material/styles";
+import Tooltip, { TooltipProps, tooltipClasses } from "@mui/material/Tooltip";
+import HelpOutlineIcon from "@mui/icons-material/HelpOutline";
+import { useTheme } from "@mui/material";
+
+const CustomizedTooltip = styled(
+  ({ className, placement, arrow, ...props }: TooltipProps) => (
```

Comment:
I believe all you need to do to convert this from Typescript to Javascript is remove this `TooltipProps` type.